Batteries/Data/Vector/Lemmas.lean

```diff
@@ -41,6 +41,26 @@ theorem toArray_injective : ∀ {v w : Vector α n}, v.toArray = w.toArray → v
 @[deprecated (since := "2024-11-25")] alias toArray_swapN := toArray_swap
 @[deprecated (since := "2024-11-25")] alias toArray_swapAtN := toArray_swapAt
 
+/-! ### tail lemmas -/
+
+theorem tail_zero {v : Vector α 0} : v.tail = #v[] := Vector.eq_empty
+
+theorem tail_ne_zero [NeZero n] {v : Vector α n} :
+    v.tail = v.eraseIdx 0 n.pos_of_neZero := dif_pos _
```

Comment:
This looks misnamed to me; it should be `tail_eq_eraseIdx` or `tail_eq_of_neZero`

Reply:
I think I agree but unfortunately we have now merged.

Reply:
you can submit a follow up PR

Reply:
Oh if that's ok for something small, I'll do it tomorrow.

Reply:
#1236